### DISCUSSION TITLE
حل مشکل کانکشن مستقیم ایرانسل

### DISCUSSION THREAD

**Discussion by amirbayat82** [Gozargah/Marzban#816] [Ideas]
با سلام خدمت دوستان عزیز
مطالب اینجا خیلی مفید بود و واقعا استفاده کردم . گفتم منم به نوبه خودم یه کمک کوچیک برسونم
قضیه از این قراره که ایرانسل از هفته گذشته اختلال شدیدی روی کانکشن های مستقیم انداخته . که بر خلاف تصور دوستان ؛ اختلال روی کانفیگ های TLS دار هست و اختلال روی رنج آیپی خاصی نیست ( به استثنا کلودفلر)
خب بازم خیلی ها با این وضعیت همچنان نمیخوان تانل بزنن حالا باید چیکار کنن ؟
این کار ممکنه نیاز به تعویض آیپی داشته باشه اما واسه چند روز یا حتی هفته جوابه و روش خوبیه برای فرار از تانل
کافیه یه کانفیگ ریلیتی grpc با Sni سایت greenpepper.ir بسازید . سرعتش عالیه و فعلا هم داره جواب میده
موفق و پیروز باشید

**Comment by abedin70** [comment]
چند وقت از این روش استفاده کردی؟
سرور فیلتر نشده؟

**Reply by amirbayat82** [reply]
ببینید اگر میخواین کانکشن مستقیم برقرار کنید در هر صورتی خطر فیلترینگ شما رو تهدید میکنه . برای کسایی خوبه که میتونن سریع چنج آیپی کنن و گرنه اینم بعد چند روز زده میشه
